shell/plugins/trim-whitespace.js

```diff
@@ -14,22 +14,8 @@ export function trimWhitespace(el, dir) {
   }
 }
 
-export function trimWhitespaceSsr(el, dir) {
-  // This causes server<->client dom mismatches sometimes... gave up for now.
-  /*
-  for ( const node of (el.children || []) ) {
-    if ( node.text ) {
-      const trimmed = node.text.trim();
-
-      if ( trimmed !== node.text ) {
-        node.text = trimmed;
-      }
-    } else if ( node.children ) {
-      trimWhitespaceSsr(node);
-    }
-  }
-  */
-}
+/* eslint-disable-next-line no-console */
+console.warn('The implicit addition of trim-whitespace has been deprecated in Rancher Shell and will be removed in a future version. Make sure to invoke `Vue.use(trimWhitespaceDirective)` to maintain compatibility.');
```

Comment:
I guess it would be `Vue.directive()` for these messages?

```suggestion
console.warn('The implicit addition of trim-whitespace has been deprecated in Rancher Shell and will be removed in a future version. Make sure to invoke `Vue.directive(trimWhitespaceDirective)` to maintain compatibility.');
```